projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/DockerRegistryService.java

```diff
@@ -5,10 +5,26 @@
 
 package com.vmware.taurus.service.deploy;
 
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.auth.BasicSessionCredentials;
+import com.amazonaws.services.ecr.model.ImageNotFoundException;
+import com.amazonaws.services.ecr.model.InvalidParameterException;
+import com.amazonaws.services.ecr.model.RepositoryNotFoundException;
+import com.amazonaws.services.ecr.model.ServerException;
+import com.vmware.taurus.service.credentials.AWSCredentialsService;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 
+import com.amazonaws.services.ecr.AmazonECR;
+import com.amazonaws.services.ecr.AmazonECRClientBuilder;
+import com.amazonaws.services.ecr.model.DescribeImagesRequest;
+import com.amazonaws.services.ecr.model.DescribeImagesResult;
+import com.amazonaws.services.ecr.model.ImageIdentifier;
+
 @Service
+@Slf4j
 public class DockerRegistryService {
```

Comment:
Do we have unit tests?

Reply:
I believe the projects/control-service/projects/pipelines_control_service/src/test/java/com/vmware/taurus/service/deploy/JobImageBuilderTest.java tests.